.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "oauth2-proxy Dev",
+  "context": "..",
+  "dockerFile": "./Dockerfile",
+  "postCreateCommand": "pre-commit install && go mod download",
+  "containerEnv": { "DEVCONTAINER": "1" },
+  "appPort": ["4180:4180"],
+  "runArgs": ["-e", "GIT_EDITOR=code --wait"],
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "golang.go",
+        "ms-vscode.makefile-tools",
+        "ms-azuretools.vscode-docker",
```

Comment:
This is not strictly necessary as it will already be installed by the docker-in-docker feature

Reply:
https://github.com/devcontainers/features/blob/main/src/docker-in-docker/devcontainer-feature.json